Weather/Weather/Data/Model/Coordinate.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Coordinate.swift
+//  Weather
+//
+//  Created by minh duc on 2/21/19.
+//  Copyright © 2019 minhduc. All rights reserved.
+//
+
+import ObjectMapper
+
+struct Coordinate: Mappable {
+    var lat = 0.0
+    var lng = 0.0
```

Comment:
nếu là số với kiểu dữ liệu khác Int thì phải định nghĩa rõ cả kiểu dữ liệu
VD:
```suggestion
    var lng: CGFloat = 0.0
```